src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/prevalue/blocklist.blockconfiguration.controller.js

```diff
@@ -218,7 +217,15 @@
 
                 });
             } else {
-                alert("Cannot be edited cause ElementType does not exist.");
+
+                const overlay = {
+                  content: "Cannot be edited cause ElementType does not exist.",
```

Comment:
@bjarnef not sure what I did to mess up the previous review... Just wanted to highlight this text should really be localized 😄

Reply:
@nathanwoulfe yes, that would probably be a good idea although a large part on the text in settings section isn't localized.

I may have some time to update that otherwise feel free to merge this and I can submit a new PR to localize the text.

Reply:
All good - if there's more text, makes sense to keep it in a separate PR. 

Will get to this one today 👍